src/index.css

```diff
@@ -15,19 +15,21 @@
 }
 
 :root {
-  /* Colors */
-  --color-background: #fff4ec;
-  --color-txt: #45312e;
-  --color-btn: #f27d88;
+  --color-background: #fffffa;
+  --color-txt: #333138;
+  --color-btn: #d83c56;
   --color-btn-hover: #fa9da6;
-  --color-action-btn: #ff3264;
-  --color-action-btn-hover: #ff668c;
-  --color-header-footer: #ff9671;
+  --color-search-btn: #d90429;
+  --color-search-btn-hover: #fa7885;
+  --color-action-btn: #d90429;
+  --color-action-btn-hover: #fa7885;
+  --color-header-footer: #333138;
   --color-white: #fff;
-  --color-black: #000;
+  --color-black: #0000;
```

Comment:
You don't need to change this, but use 6 digit or 3-digit HEX code. to keep it consistent